tests/test_manager_edgestore.py

```diff
@@ -0,0 +1,624 @@
+import os
+import tempfile
+from collections import Counter
+
+import pybel
+from pybel.constants import *
+from pybel.manager import models, Manager
+from pybel.utils import hash_dump
+from tests import constants
+from tests.constants import (
+    BelReconstitutionMixin,
+    test_bel_simple,
+    test_bel_simple_b,
+    expected_test_simple_metadata,
+)
+from tests.mocks import mock_bel_resources
+
+
+class TestEdgeStore(BelReconstitutionMixin):
+    """Tests that the cache can be queried"""
+
+    def setUp(self):
+        self.test_connection = os.environ.get('PYBEL_TEST_CONNECTION')
+
+        if self.test_connection:
+            self.connection = self.test_connection
+        else:
+            self.fd, self.path = tempfile.mkstemp()
+            self.connection = 'sqlite:///' + self.path
+
+        log.info('Test generated connection string %s', self.connection)
+
+        self.manager = Manager(connection=self.connection)
+        self.manager.create_all()
+
+        self.setup_graph_and_network()
+
+    def tearDown(self):
+        # Commiting the session to close the reading transaction (metadata lock)
+        self.manager.session.commit()
+        self.manager.drop_all()
+        self.manager.session.close()
+
+        if not self.test_connection:
+            os.close(self.fd)
+            os.remove(self.path)
+
+    @mock_bel_resources
+    def setup_graph_and_network(self, mock_get):
+        self.graph = pybel.from_path(path=test_bel_simple, manager=self.manager, allow_nested=True)
+        self.network = self.manager.insert_graph(self.graph)
```

Comment:
It would be better to take advantage of the pre-coded networks in the examples module so your tests don;t have to go through the overhead of parsing the BEL files